src/oogeso/core/devices/sink.py

```diff
@@ -12,6 +12,10 @@ def getFlowVar(self, pyomo_model, t):
         return pyomo_model.varDeviceFlow[self.id, "el", "in", t]
 
 
+# Just another name (to mirror Powersource):
+Powersink = Sink_el
```

Comment:
Should rename the original class to PowerSink and make an inherited subclass. Naming is PascalCase for classes. I.e ElSink and PowerSink. No underscore :)

PowerSink(Device):
   ...

class ElSink(PowerSink): 
    pass:

Reply:
Done the subclass thing. Naming changes require changes elsewhere so I'll leave it for now.